_data/trans/hi.json

```diff
@@ -0,0 +1,47 @@
+{
+    "about": "हमारे बारे मे",
+    "contribute": "GitHub पर योगदान दे |",
+    "defaultnote_easy": "दिए हुए पेज पर जाए और अपना अकाउंट मिटाए |",
+    "defaultnote_email": "दिए गए पते पर अकाउंट मिटाने के लिए ईमेल करे |",
+    "difficulty": "स्तर",
+    "difficulty_easy": "आसान",
+    "difficulty_hard": "कठिन",
+    "difficulty_impossible": "असंभव",
+    "difficulty_limited": "बहुत मुश्किल",
+    "difficulty_medium": "मध्यम",
+    "extension_browser": "क्रोम या फायर फॉक्स एक्सटेंशन",
+    "extensionguide": "Extesnion के बिंदुओं का मतलब",
+    "extensionp1": "जब आप ऐसी वेबसाइट का प्रयोग कर होंगे, जिसके बारे मे JustDeleteMe पर जानकारी होगी, तब यह extension \nसर्च इंजन के दाई ओर बने पट्टी पर एक छोटा सा बिंदु बना देगा | तब दिए हुए बिंदु पर क्लिक करने पर आपको अकाउंट मिटाने का तरीका पता चलेगा |",
+    "extensionp2": "Extension इस्तेमाल करने के लिए जाए",
+    "extensionp3": "क्रोम वेब स्टोर (अनाधिकारिक)",
+    "extensionp4": "फायर फॉक्स का टूल",
+    "extensionp5": "",
+    "footercredits": "निर्माता",
+    "guide": "दिशानिर्देश",
+    "guideeasy": "आसान प्रक्रिया",
+    "guideexplanations": "ऊपर दिए गए लिंक पर एक रंग बिंदु है, जो अकाउंट मिटाने करने के प्रक्रिया की कठनाई का स्तर बताता है |",
+    "guidehard": "पूरी तरह से अकाउंट मिटने के लिए ग्राहक सेवा केन्द्र से संपर्क करना पड़ेगा",
+    "guideimpossible": "मिटाया नहीं जा सकता है",
+    "guidelimited": "अगर आपके क्षेत्र मतलब देश या राज्य मे निजता के अधिकार लागू होंगे, तभी आप अकाउंट मिटा सकते है",
+    "guidemedium": "कुछ और प्रक्रियाएं पूरी करनी होंगी",
+    "hideinfo": "जानकारी छुपाए...",
+    "jgmd": "अपना डाटा पाने के लिए जाए",
+    "name": "English",
+    "noinfo": "No Info Available",
+    "noresults": "जरूरी जानकारी नहीं मिल रही है?",
+    "noresultshelp": "JustDeleteMe को सुधारने मे मदद करे",
+    "popular": "Popular",
+    "pullrequest": "GitHub पर एक Pull Request करे",
+    "reset": "Reset",
+    "search": "Search",
+    "sendmail": "ईमेल भेजे",
+    "showinfo": "जानकारी दिखाए",
+    "tagline": "इंटरनेट की के अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
+    "title": "Just Delete Me | इंटरनेट के कई अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
+    "twitter": "JDM के बारे मे ट्वीट करे",
+    "whatisthis": "यह क्या है?",
+    "whatisthis1": "Many companies use",
+    "whatisthis2": "dark pattern",
```

Comment:
This is the beginning of the phrase 

>"Many companies use [dark pattern](https://darkpatterns.org/) techniques to make it difficult to find how to delete your account. JustDeleteMe aims to be a directory of urls to enable you to easily delete your account from web services."

@shivam-maurya-git sure it should not be translated?

Reply:
I forgotten to save translation, now I saved it. Now, it is updated.